ERROR - 2024-11-11 18:54:34 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#11 {main}
CRITICAL - 2024-11-11 18:54:34 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: POST, Route: loginCheck]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
 3 APPPATH/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb()
10 SYSTEMPATH/rewrite.php(44): require_once('/home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php')
ERROR - 2024-11-11 19:05:40 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#11 {main}
CRITICAL - 2024-11-11 19:05:40 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: POST, Route: loginCheck]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
 3 APPPATH/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb()
10 SYSTEMPATH/rewrite.php(44): require_once('/home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php')
ERROR - 2024-11-11 19:06:08 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#11 {main}
CRITICAL - 2024-11-11 19:06:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: POST, Route: loginCheck]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
 3 APPPATH/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb()
10 SYSTEMPATH/rewrite.php(44): require_once('/home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php')
ERROR - 2024-11-11 19:06:10 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#11 {main}
CRITICAL - 2024-11-11 19:06:10 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: POST, Route: loginCheck]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
 3 APPPATH/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb()
10 SYSTEMPATH/rewrite.php(44): require_once('/home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php')
ERROR - 2024-11-11 19:06:29 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#11 {main}
CRITICAL - 2024-11-11 19:06:29 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: POST, Route: loginCheck]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
 3 APPPATH/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb()
10 SYSTEMPATH/rewrite.php(44): require_once('/home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php')
ERROR - 2024-11-11 19:07:24 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#11 {main}
CRITICAL - 2024-11-11 19:07:24 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: POST, Route: loginCheck]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
 3 APPPATH/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb()
10 SYSTEMPATH/rewrite.php(44): require_once('/home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php')
ERROR - 2024-11-11 19:07:26 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#11 {main}
CRITICAL - 2024-11-11 19:07:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: POST, Route: loginCheck]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
 3 APPPATH/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb()
10 SYSTEMPATH/rewrite.php(44): require_once('/home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php')
ERROR - 2024-11-11 19:11:31 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/app/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(325): CodeIgniter\CodeIgniter->run()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php(56): CodeIgniter\Boot::bootWeb()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/rewrite.php(44): require_once('...')
#11 {main}
CRITICAL - 2024-11-11 19:11:31 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: POST, Route: loginCheck]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/BaseBuilder.php(1782): CodeIgniter\Database\BaseConnection->query()
 3 APPPATH/Controllers/Auth.php(22): CodeIgniter\Database\BaseBuilder->getWhere()
 4 SYSTEMPATH/CodeIgniter.php(933): App\Controllers\Auth->logincheck()
 5 SYSTEMPATH/CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController()
 6 SYSTEMPATH/CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest()
 7 SYSTEMPATH/Boot.php(325): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH/Boot.php(67): CodeIgniter\Boot::runCodeIgniter()
 9 FCPATH/index.php(56): CodeIgniter\Boot::bootWeb()
10 SYSTEMPATH/rewrite.php(44): require_once('/home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/public/index.php')
ERROR - 2024-11-11 19:15:56 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CLI\Commands->run()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(351): CodeIgniter\CLI\Console->run()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(104): CodeIgniter\Boot::runCommand()
#12 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CLI\Commands->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(351): CodeIgniter\CLI\Console->run()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(104): CodeIgniter\Boot::runCommand()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-11 19:19:46 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CLI\Commands->run()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(351): CodeIgniter\CLI\Console->run()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(104): CodeIgniter\Boot::runCommand()
#12 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CLI\Commands->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(351): CodeIgniter\CLI\Console->run()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(104): CodeIgniter\Boot::runCommand()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/spark(84): CodeIgniter\Boot::bootSpark()
#12 {main}
ERROR - 2024-11-11 19:19:54 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:187
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(187): mysqli->real_connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/Database.php(72): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/Config.php(111): CodeIgniter\Database\Database->loadForge()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/Seeder.php(98): CodeIgniter\Database\Config::forge()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Commands/Database/Seed.php(71): CodeIgniter\Database\Seeder->__construct()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CLI\Commands->run()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(351): CodeIgniter\CLI\Console->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(104): CodeIgniter\Boot::runCommand()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/spark(84): CodeIgniter\Boot::bootSpark()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:231
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/Database.php(72): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/Config.php(111): CodeIgniter\Database\Database->loadForge()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/Seeder.php(98): CodeIgniter\Database\Config::forge()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Commands/Database/Seed.php(71): CodeIgniter\Database\Seeder->__construct()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CLI\Commands->run()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(351): CodeIgniter\CLI\Console->run()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(104): CodeIgniter\Boot::runCommand()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/spark(84): CodeIgniter\Boot::bootSpark()
#10 {main}
CRITICAL - 2024-11-11 19:19:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
[Method: CLI, Route: db:seed]
in SYSTEMPATH/Database/BaseConnection.php on line 467.
 1 SYSTEMPATH/Database/Database.php(72): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/Database/Config.php(111): CodeIgniter\Database\Database->loadForge()
 3 SYSTEMPATH/Database/Seeder.php(98): CodeIgniter\Database\Config::forge()
 4 SYSTEMPATH/Commands/Database/Seed.php(71): CodeIgniter\Database\Seeder->__construct()
 5 SYSTEMPATH/CLI/Commands.php(70): CodeIgniter\Commands\Database\Seed->run()
 6 SYSTEMPATH/CLI/Console.php(48): CodeIgniter\CLI\Commands->run()
 7 SYSTEMPATH/Boot.php(351): CodeIgniter\CLI\Console->run()
 8 SYSTEMPATH/Boot.php(104): CodeIgniter\Boot::runCommand()
 9 ROOTPATH/spark(84): CodeIgniter\Boot::bootSpark()
CRITICAL - 2024-11-11 19:21:27 --> ErrorException: Cannot use empty array elements in arrays
[Method: CLI, Route: migrate]
in APPPATH/Config/Database.php on line 32.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-11 19:21:33 --> ErrorException: Cannot use empty array elements in arrays
[Method: CLI, Route: db:seed]
in APPPATH/Config/Database.php on line 32.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-11 19:22:11 --> ErrorException: Cannot use empty array elements in arrays
[Method: CLI, Route: db:seed]
in APPPATH/Config/Database.php on line 32.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-11-11 19:22:12 --> ErrorException: Cannot use empty array elements in arrays
[Method: CLI, Route: migrate]
in APPPATH/Config/Database.php on line 32.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-11-11 19:23:45 --> Error connecting to the database: Error: Class "SQLite3" not found in /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/SQLite3/Connection.php:93
Stack trace:
#0 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(421): CodeIgniter\Database\SQLite3\Connection->connect()
#1 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/SQLite3/Connection.php(63): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(616): CodeIgniter\Database\SQLite3\Connection->initialize()
#3 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1485): CodeIgniter\Database\BaseConnection->query()
#4 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/BaseConnection.php(1505): CodeIgniter\Database\BaseConnection->listTables()
#5 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(759): CodeIgniter\Database\BaseConnection->tableExists()
#6 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(163): CodeIgniter\Database\MigrationRunner->ensureTable()
#7 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(85): CodeIgniter\Database\MigrationRunner->latest()
#8 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Commands.php(70): CodeIgniter\Commands\Database\Migrate->run()
#9 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CLI\Commands->run()
#10 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(351): CodeIgniter\CLI\Console->run()
#11 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/vendor/codeigniter4/framework/system/Boot.php(104): CodeIgniter\Boot::runCommand()
#12 /home/nikunj/Desktop/Axita/lunoz/lunoz_codeigniter/spark(84): CodeIgniter\Boot::bootSpark()
#13 {main}
